src/test/java/org/smartregister/helpers/LocationHelperTest.java

```diff
@@ -0,0 +1,45 @@
+package org.smartregister.helpers;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.Location;
+import org.junit.Assert;
+import org.junit.Test;
+import org.smartregister.utils.Constants;
+
+public class LocationHelperTest {
+
+  @Test
+  public void testUpdateLocationTags() {
```

Comment:
I think the method we should be targeting for unit testing is `LocationHelper.updateLocationLineage`